### PR TITLE
testutils: reduce dependency on testify package in testutils

### PR DESCRIPTION
**Pull request by thaJeztah**

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @tiborvass @cyli @mavenugo PTAL

File: server/handlers/utils_test.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/theupdateframework/notary/tuf/data"
+	"github.com/theupdateframework/notary/tuf/signed"
+	"github.com/theupdateframework/notary/tuf/testutils"
+)
+
+func mustCopyKeys(t *testing.T, from signed.CryptoService, roles ...data.RoleName) signed.CryptoService {
+	cs, err := mustCopyKeys(t, from, roles...)
```
Comment (justincormack) [inline server/handlers/utils_test.go]: shouldnt that be `testutils.CopyKeys`?

Comment (thaJeztah) [reply server/handlers/utils_test.go]: Ah, dang; you're right. Must've done find/replace _after_ I added that file 🤦‍♂️ and forgot to update. Fixing now

**Comment by thaJeztah** [comment]
@justincormack fixed; ptal

File: tuf/testutils/repo.go [hunk 2]
```diff
@@ -54,16 +52,18 @@ func CreateKey(cs signed.CryptoService, gun data.GUN, role data.RoleName, keyAlg
 }
 
 // CopyKeys copies keys of a particular role to a new cryptoservice, and returns that cryptoservice
-func CopyKeys(t *testing.T, from signed.CryptoService, roles ...data.RoleName) signed.CryptoService {
+func CopyKeys(t *testing.T, from signed.CryptoService, roles ...data.RoleName) (signed.CryptoService, error) {
```
Comment (cyli) [inline tuf/testutils/repo.go]: Would it make sense to remove the `t *testing.T` requirement from the function signature since it's no longer being used?

**Comment by justincormack** [comment]
If you rebase CI should pass (although about to run into other circle ci issues)

File: tuf/testutils/repo.go (hunk 2 above)
Comment (thaJeztah) [reply tuf/testutils/repo.go]: Yes, looks like that would make sense; I can update

**Comment by thaJeztah** [comment]
Rebased, and made the suggested changes

**Comment by justincormack** [comment]
@thaJeztah it seems to need a rebase...

**Comment by thaJeztah** [comment]
rebased

**Comment by cyli** [comment]
Still +1 :)

**Pull request merged**